lib/ebooks/twitter.rb

```diff
@@ -18,9 +18,34 @@ def twitter_client
     end
 
     def tweet(tweet_text)
-      tweet_text = tweet_text.gsub('@', '')[0..139]
+      tweet_text = truncate tweet_text.gsub('@', '')
```

Comment:
I have now fully-qualified the call. I really want to wrap some tests around all of this...